src/SDO.cpp

```diff
@@ -20,7 +20,7 @@ canbus::Message canopen_master::makeSDOInitiateDomainUpload(uint8_t nodeId, uint
 
 canbus::Message canopen_master::makeSDOInitiateDomainDownload(
     uint8_t nodeId, uint16_t objectIndex, uint8_t objectSubindex,
-    uint8_t const* data, uint32_t size
+    uint8_t const* data, uint32_t size, bool s
```

Comment:
please rename the flag to something more explicit, e.g. `sizeInData`

Reply:
:+1: